### DISCUSSION TITLE
Request chart of the generation speed of tokens (tokens/s)

### DISCUSSION THREAD

**Discussion by secsilm** [langfuse/langfuse#1769] [Ideas]
### Describe the feature or potential improvement

Like this:

![image](https://github.com/langfuse/langfuse/assets/8894643/098c1afb-bdca-4a8c-9f41-a12da0572234)

**Comment by clemra** [comment]
Thank you, that makes sense. We will add it to our backlog. We will revamp the dashboards soon.
